homeassistant/components/auth/__init__.py

```diff
@@ -196,24 +196,26 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     hass.http.register_view(TokenView(retrieve_result))
     hass.http.register_view(LinkUserView(retrieve_result))
 
-    hass.components.websocket_api.async_register_command(
-        WS_TYPE_CURRENT_USER, websocket_current_user, SCHEMA_WS_CURRENT_USER
+    websocket_api.async_register_command(
+        hass, WS_TYPE_CURRENT_USER, websocket_current_user, SCHEMA_WS_CURRENT_USER
```

Comment:
Another cleanup that we can do for websockets is to migrate to the command decorator to attach type/schema.